src/utils/prometheus.js

```diff
@@ -0,0 +1,16 @@
+const apiMetrics = require('prometheus-api-metrics');
+const { Configuration } = require('@schul-cloud/commons');
+
+module.exports = (app) => {
+	if (Configuration.has('FEATURE_PROMETHEUS_ENABLED') && Configuration.get('FEATURE_PROMETHEUS_ENABLED') === true) {
+		const metricsOptions = {};
+		if (Configuration.has('PROMETHEUS__METRICS_PATH')) {
```

Comment:
Where is PROMETHEUS__METRICS_PATH define?

Reply:
this is in https://github.com/hpi-schul-cloud/schulcloud-server/pull/1839/files#diff-acaf4b97e02a99f90554f68ab6274b11b7c59c56cd025fdc6c9bfa146731c6e1R401